src/main/java/salesforce/ui/pages/product/NewProductPage.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package salesforce.ui.pages.product;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.entities.Product;
+import salesforce.ui.pages.BasePage;
+import salesforce.utils.Translator;
+import java.util.HashMap;
+import java.util.Set;
+
+/**
+ * This class returns an instance of NewProductForm.
+ */
+public class NewProductPage extends BasePage {
+    @FindBy(css = ".forceActionButton:nth-child(3) > .label")
```

Comment:
I recommend to search for another locator, that doesn't use indexing, for next PR maybe you could look for another way

Reply:
Thanks. I will change it for the next PR